packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -320,11 +326,9 @@ exports.onCreateWebpackConfig = ({ actions, plugins }, themeOptions) => {
 };
 
 exports.onPostBootstrap = (_, themeOptions) => {
-  const { refetch, file } = themeOptions;
-
-  if (refetch) {
+  if (themeOptions.swiftype && themeOptions.swiftype.refetch) {
```

Comment:
If you hate typing `themeOptions.swiftype` over and over, you should be able to use a combo of destructuring and defaults to make this a bit easier to parse. Up to you if you want to do this:

```js
const { swiftype = {} } = themeOptions

if (swiftype.refetch) {
  fs.writeFileSync(swiftype.file, ....)
}
```